data-node/src/main/java/org/graylog/datanode/build/InstallOpensearchPlugins.java

```diff
@@ -0,0 +1,89 @@
+/*
+ * Copyright (C) 2020 Graylog, Inc.
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the Server Side Public License, version 1,
+ * as published by MongoDB, Inc.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * Server Side Public License for more details.
+ *
+ * You should have received a copy of the Server Side Public License
+ * along with this program. If not, see
+ * <http://www.mongodb.com/licensing/server-side-public-license>.
+ */
+package org.graylog.datanode.build;
+
+import org.graylog.datanode.opensearch.cli.CliEnv;
+import org.graylog.datanode.opensearch.cli.OpensearchCli;
+
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.util.Arrays;
+import java.util.List;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+/**
+ * Utility for batch installation of opensearch plugins from maven builds.
+ */
+public class InstallOpensearchPlugins {
+
+    public static final Pattern PLUGIN_NAME_PATTERN = Pattern.compile("^(.*)-\\d+\\.\\d+\\.\\d+\\.zip$");
+
+    public static void main(String[] args) throws IOException {
+
+        if (args.length < 2) {
+            System.err.println("Syntax: " + InstallOpensearchPlugins.class.getSimpleName() + " <opendsearchDistDir> <pluginFile1> ... <pluginFileN>");
+            System.exit(1);
+        }
+
+        final Path opensearchDist = Path.of(args[0]);
+        final Path binDir = opensearchDist.resolve("bin");
+        final Path configDir = opensearchDist.resolve("config");
+
+        final CliEnv env = new CliEnv(configDir).javaHome(System.getProperty("java.home"));
```

Comment:
shouldn't this be `opensearchDist.resolve("jdk") and the system jdk only on Mac?`

Reply:
When we don't override the OPENSEARCH_JAVA_HOME then opensearch will by default use the bundled version. That's how it worked before and how is it working when triggering CLI tools during datanode runtime.

In the build we have to avoid using distributed JDK, as this won't match our architecture (aarch64 vs x64), so we are forcing opensearch to use our own JDK by providing the explicit env setting. 

Here we had no special handling for mac, so I am not sure if we need it now. Maybe we do?

Reply:
you are right. sorry, i missed that line in the pom.xml before the change